supervisor/api/addons.py

```diff
@@ -379,8 +379,10 @@ async def options_config(self, request: web.Request) -> None:
         slug: str = request.match_info.get("addon")
         if slug != "self":
             raise APIForbidden("This can be only read by the Add-on itself!")
-
         addon = self._extract_addon_installed(request)
+
+        # Lookup/reload secrets
+        await self.sys_homeassistant.secrets.reload()
```

Comment:
This method is ratelimited <https://github.com/home-assistant/supervisor/blob/main/supervisor/homeassistant/secrets.py#L44>
Do we need some flag to bypass that limit?

Reply:
I think that is fine, so we avoid useless update secrets?

Reply:
But what's the point then?
Your description states "Make sure, secrets are up-to-date" this will not be true, secrets can change and not be updated here until the rate limit has lifted

Reply:
I have update the description